docs/jax-101/02-jitting.md

```diff
@@ -0,0 +1,293 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.0
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
++++ {"id": "O-SkdlPxvETZ"}
+
+# How does JAX work?
```

Comment:
"Just-In-Time compiling with JAX" ?

Reply:
@skye I noticed that there's a _"Using `jax.jit`"_-like section title in the [JAX QuickStart](https://jax.readthedocs.io/en/latest/notebooks/quickstart.html#using-jit-to-speed-up-functions) called _"Using `jit()` to speed up functions"_. 

Also, that JAX QuickStart refers to all the cool APIs with round brackets (`jit()`), which is consistent with the official [Style Guide for Python](https://www.python.org/dev/peps/pep-0008/) (e.g. _"For example, the os.stat() function returns a tuple"_).

Reply:
That comes from using RST directives to refer to functions. We could do that here (e.g. ```{func}`jax.jit` ``` in markdown) and it would render the text and links correctly in the HTML docs, but you'd still see the raw markdown in the notebooks.

It's another case of difficulty introduced by the fact that we want this single source to operate in two ways: as a viewable notebook, and as a built documentation page.